src/Utils/Image.php

```diff
@@ -90,8 +90,8 @@
  * @method void stringUp($font, $x, $y, string $s, $col)
  * @method void trueColorToPalette(bool $dither, $ncolors)
  * @method array ttfText($size, $angle, $x, $y, $color, string $fontfile, string $text)
- * @property-read int $width
- * @property-read int $height
+ * @property-read positive-int $width
+ * @property-read positive-int $height
```

Comment:
What's it good for?

Reply:
Mostly for compatibility with other classes that already expect width and height to be >= 0. e.g. for database entity.

Reply:
As any type (narrowing) hint, it's good to make code more readable and eliminate useless checks.

If you rely on static analysis and narrow down the types enough, [checks like this](https://github.com/nette/utils/pull/290/files#diff-94b775b0ed8e491ffeeccac27715ea411933e493574562c3ff790577661ab58fR233) become useless and can be safely eliminated.

Dimensions are often used in various calculations. Knowing that those are actually positive integers, implies e.g.:
- their ratio is well defined and also positive number
- the area (number of pixels in image) is positive integer